include/sys/device.h

```diff
@@ -26,6 +26,7 @@ typedef int (*d_detach_t)(device_t *dev);
 typedef enum {
   DIF_BUS,
   DIF_PCI_BUS,
+  DIF_ICU,  /* Interrupt Controller Unit */
```

Comment:
More commonly used name for that is _Programmable Interrupt Controller_ (PIC for short). 
https://en.wikipedia.org/wiki/Programmable_interrupt_controller